TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/device/usb/UsbDeviceConnection.java

```diff
@@ -71,7 +72,7 @@ public void onReceive(String data) {
 
     @Override
     public void send(String data) {
-        // TODO: send data to serial port.
+        serialPortMonitor.tryToSendCommand(data);
```

Comment:
What? how were we even sending commands? 😅

Reply:
Yea, this was the thing I found weird because I remember seeing a video where you moved the bot 😂

Reply:
I spent ages wondering why it wasn't working, only to stumble across the TODO 😂

Reply:
Lol who knows how I made it working... oh well this makes sense